quickjs.c

```diff
@@ -14826,16 +14806,7 @@ typedef enum {
 #define FUNC_RET_YIELD      1
 #define FUNC_RET_YIELD_STAR 2
 
-#if defined(DUMP_BYTECODE_FINAL) || \
-    defined(DUMP_BYTECODE_PASS2) || \
-    defined(DUMP_BYTECODE_PASS1) || \
-    defined(DUMP_BYTECODE_STACK) || \
-    defined(DUMP_BYTECODE_STEP)  || \
-    defined(DUMP_READ_OBJECT)
-#define DUMP_BYTECODE
-#endif
```

Comment:
FYI I also found this section wasn't needed.

Reply:
Yes, no longer needed, but somewhat confusing. It was used to trigger compilation of the bytecode dump utilities when one of the flags that needed it was defined.  `JS_DUMP_BYTECODE` is no longer defined anywhere so the tests `#ifdef ENABLE_DUMPS // JS_DUMP_BYTECODE`  may be misleading.

Reply:
This was something I considered. I figured readers would be able to work out what's going on. 

I changed the comments to `JS_DUMP_BYTECODE_*` to give a bit more of a hint.